Martium.GuessTheRiddleGame/Program.cs

```diff
@@ -152,11 +145,51 @@ private static void ShowPlayerResult(Dictionary<string, int> playerResult)
                 WriteColoredMessage($" { playerResults.Key} -> {playerResults.Value}", color);
             }
         }
+        private static void StartGames()
+        {
+            int gameNumber = 1;
+            string repeatGame = "y";
+
+            while(repeatGame == "y")
+            {
+                WriteAndSeperateText($"================================= GAME #{gameNumber} =================================", addNewLineBeforeText:true, addNewLineAfterText:true);
+
+                Dictionary<string, int> playerResult = PlayGame();
+
+                ShowPlayerResult(playerResult);
+
+                repeatGame = AskRepeat();
+               
+                gameNumber++;
+            }
+        }
+
+        private static void WriteAndSeperateText(string message,bool addNewLineBeforeText = false, bool addNewLineAfterText = false)
+        {
+            if (addNewLineBeforeText)
+            {
+                Console.WriteLine();
+            }
+
+                Console.WriteLine(message);
+
+            if (addNewLineAfterText)
+            {
+                Console.WriteLine();
+            }
+        }
+
+        private static string AskRepeat()
+        {
+            WriteAndSeperateText("Jei norite pakartoti žaidimą spauskite klavišą 'y' ir paspauskite ENTER", addNewLineBeforeText: true);
```

Comment:
Method could be named `WriteLine` because:  
- `And` word implies that text will be separated always, but separation is optional thing and if both variables is false no sepration will happen
- `Write` could be confused with Console.Write where text is put not in the new line but after current text in same line
- Also such name would be shorter and less text to read